### PR TITLE
replaced from repo lesson-2 to if-html

### PR DESCRIPTION
**Pull request by ilyabaranowski**

### CODE REVIEW

File: about/index.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    <meta charset="UTF-8"/>
+    <meta name="Ilya" content="Page about me"/>
+    <title>My first page</title>
+</head>
+<body>
+    <h1>My first page</h1>
+    <p><a href="./index.html"></a></p>
+    <h2>Why Front-end?</h2>
+    <span>Among the many different directions for development in the IT-field, I chose the front-end.
+    <br/>
```
Comment (ghost) [inline about/index.html]: Don't use `br` in another inline tags.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,2 @@
+    <h1>My first page</h1>
```
Comment (ghost) [inline index.html]: You forgot to add a basic structure for your file.

File: about/index.html [hunk 3]
```diff
@@ -0,0 +1,33 @@
+<html>
+<head>
+    <meta charset="UTF-8"/>
+    <meta name="Ilya" content="Page about me"/>
+    <title>My first page</title>
+</head>
+<body>
+    <h1>My first page</h1>
+    <p><a href="./index.html"></a></p>
```
Comment (ghost) [inline about/index.html]:
```suggestion
 <p><a href="../index.html"></a></p>
```

File: about/index.html (hunk 1 above)
Comment (ilyabaranowski) [reply about/index.html]: br deleted

File: index.html (hunk 2 above)
Comment (ilyabaranowski) [reply index.html]: added basic structure

File: about/index.html (hunk 3 above)
Comment (ilyabaranowski) [reply about/index.html]: done

**Pull request merged**